cmd/root.go

```diff
@@ -172,6 +172,8 @@ the maximum time has passed. Defaults to 0s.`)
 		"Enable Prometheus for metric collection using the provided namespace")
 	cmd.PersistentFlags().StringVar(&c.httpPort, "http-port", "9090",
 		"Port for the Prometheus server to use")
+	cmd.PersistentFlags().StringVar(&c.conf.QuotaProject, "quota-project", "",
+		"Specifies the project to use for Cloud SQL Admin API quota tracking.\n The request IAM user must have a role with \"serviceusage.services.use\" \nSee https://cloud.google.com/service-usage/docs/overview and https://cloud.google.com/storage/docs/requester-pays")
```

Comment:
I'll make a final pass on all these descriptions to make sure they render well on smaller screens. In the meantime, let's use backticks to make the formatting here a little cleaner (also updating the wording a bit here):

``` golang
               `Specifies the project to use for Cloud SQL Admin API quota tracking.
The IAM principal must have the "serviceusage.services.use" permission
for the given project. See https://cloud.google.com/service-usage/docs/overview and 
https://cloud.google.com/storage/docs/requester-pays`
```

Reply:
changed again.